tests/lwip/Makefile

```diff
@@ -1,4 +1,5 @@
 APPLICATION = lwip
+include ../Makefile.tests_common
```

Comment:
@miri64 you favored the solution of putting the prefix `tests_` into `Makefile.tests_common` which is required to avoid name clash as described in #6326 on macOS with tests where APPLICATION is equal to a required module or package name